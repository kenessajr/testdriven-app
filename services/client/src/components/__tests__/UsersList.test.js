import React from 'react'
import { shallow } from 'enzyme'

import UsersList from '../UsersList'

const users = [
    {
      'active': true,
      'email': 'kenessajr@gmail.com',
      'id': 1,
      'username': 'kenessajr'
    },
    {
      'active': true,
      'email': 'kenessa@kenessa.io',
      'id': 2,
      'username': 'kenessa'
    }
  ]

test('UsersList render properly', () => {
  const wrapper = shallow(<UsersList users={users}/>)
  const element = wrapper.find('h4')
  expect(element.length).toBe(2)
  expect(element.get(0).props.children).toBe('kenessajr')
})
