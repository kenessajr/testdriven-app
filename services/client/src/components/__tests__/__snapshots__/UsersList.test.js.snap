// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList render a snapshot properly 1`] = `
<div>
  <h4
    className="box title is-4"
  >
    kenessajr
  </h4>
  <h4
    className="box title is-4"
  >
    kenessa
  </h4>
</div>
`;
